test/mpi/test_memhooks.c

```diff
@@ -142,40 +146,46 @@ void* open_dyn_lib(const char *lib_path)
     return dl;
 }
 
-void *event_init(const char *path, ucm_mmap_hook_mode_t mmap_mode)
+void *event_init(const char *ucs_path, const char *ucm_path,
+                 ucm_mmap_hook_mode_t mmap_mode)
 {
     ucs_status_t status;
     void *dl_ucm;
 
-    dl_ucm = open_dyn_lib(path);
+    dl_ucm = open_dyn_lib(ucm_path, 0);
     if (dl_ucm == NULL) {
         return NULL;
     }
 
     status = init_ucm_config(dl_ucm, 1, mmap_mode);
     CHKERR_JUMP(status != UCS_OK, "Failed to initialize UCM", fail);
 
+    open_dyn_lib(ucs_path, RTLD_GLOBAL);
```

Comment:
No need to check rc here? Same in other similar places

Reply:
the test would fail another way if failed to load UCS (also the test is intended to test UCM, not UCS, so if UCS loading fails but test passes it's still OK)